tests/rt_utils.sh

```diff
@@ -387,13 +387,13 @@ rocoto_create_compile_task() {
     echo "  </metatask>" >> $ROCOTO_XML
   fi
 
-  # serialize WW3 builds. FIXME
+  ## serialize WW3 builds. FIXME
   DEP_STRING=""
-  if [[ ${COMPILE_PREV_WW3_NR} != '' ]]; then
-      if [[ ${MAKE_OPT^^} =~ "-DAPP=ATMW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=S2SW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFSW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFS-ALL" ]] ; then
-          DEP_STRING="<dependency><taskdep task=\"compile_${COMPILE_PREV_WW3_NR}\"/></dependency>"
-      fi
-  fi
+  #if [[ ${COMPILE_PREV_WW3_NR} != '' ]]; then
+  #    if [[ ${MAKE_OPT^^} =~ "-DAPP=ATMW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=S2SW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFSW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFS-ALL" ]] ; then
+  #        DEP_STRING="<dependency><taskdep task=\"compile_${COMPILE_PREV_WW3_NR}\"/></dependency>"
+  #    fi
+  #fi
```

Comment:
Delete commented code. `DEP_STRING` can also be removed from lines 391 and 422

Reply:
done